### ISSUE TITLE
Pagination Active slide css class not being applied.

### ISSUE THREAD

**Issue by ctesene** [zefoy/ngx-swiper-wrapper#48]
I've tried to add

```
this.swipeOptions = {
  pagination: ".swiper-pagination",
  bulletActiveClass: "swiper-pagination-bullet-active",
  paginationHide: false
};
```

but I can't get the active pagination bullet to have a css class applied so I can style it appropriately.

Anyone have any ideas?

![image](https://user-images.githubusercontent.com/1243453/29235016-270c8d68-7eb0-11e7-93c5-668ea375165e.png)

**Comment by sconix** [comment]
Have you tried without the bulletActiveClass so that Swiper uses the default class? The component provides it own custom bullet renderer as default so there might be a bug with that it wont work with custom active class. I can check that tomorrow if thats the case and release a fix. Other than that I have no idea what could cause it since it works in the example app.

Do you only have that one Swiper and no global configuration?

**Comment by ctesene** [comment]
Thanks @sconix I checked the example app, and it looks like it works. Let me see what I broke :)

**Comment by sconix** [comment]
Let me know what you find out :) Who knows if its something that can be avoided within the library.

**Comment by ctesene** [comment]
So after some troubleshooting, I found that the plugin is conflicting with [ionic2-calendar](https://github.com/twinssbc/Ionic2-Calendar). They are using Swiper as well.

If I didn't change the ```pagination``` css class, I believe the Ionic2-Calendar would select my slide pagination, and I wouldn't get an active icon. 

I changed the ```pagination``` css class in the config and my markup to something other than the default, and it all worked.

Thanks for the help!
